docs/exodus-intro.md

```diff
@@ -1,20 +1,20 @@
-# Exodus - Maven to Bazel Migration
+# Exodus - Maven to Bazel Migration 
 
 Thinking about migrating from Maven or similar to another build system? This is our story at Wix. For some more background info, take a look at our [blog articles on Medium](https://medium.com/wix-engineering/migrating-to-bazel-from-maven-or-gradle-5-crucial-questions-you-should-ask-yourself-f23ac6bca070).
 
-## Maven had been good to us...for a time
+## Maven had been good to us...for a time 
 
 Like many developers, you have probably been using Apache Maven to build your projects. We got used to managing those POM (project object model) files and waiting for our projects to build, hoping we had no errors and those pesky dependencies would all work out. 
 
 Maven was good to us when our repositories were a certain size and our projects didn't take hours to build. 
```

Comment:
size and dependency complexity.
build took hours to finish in CI (on infra code change)